Fabric.Terminology.API/Modules/ValueSetModule.cs

```diff
@@ -66,10 +69,14 @@ public sealed class ValueSetModule : TerminologyModule<ValueSetApiModel>
 
         private static async Task<T> Execute<T>(Func<Task<T>> query) => await query.Invoke();
 
-        private object GetValueSet(Guid valueSetGuid, IReadOnlyCollection<Guid> codeSystemGuids, bool summary = true)
+        private object GetValueSet(string valueSetUniqueId)
```

Comment:
Why is this input parameter a string instead of a GUID?

Reply:
It's a path argument.  I parse the Guid in the method on the following line.  TBH, I have not tried letting Nancy parse the parameter dynamically - I'll give it a go, but this seems to work well ...

Reply:
Aha. I noticed that the types switched and I didn't get why. That makes more sense.